versioned_docs/version-8.2/reference/release-policy.md

```diff
@@ -1,59 +1,111 @@
 ---
 id: release-policy
 title: "Release policy"
+description: "Learn more about Camunda releases, including alpha features and alpha releases."
 ---
 
-Camunda 8 follows the [Camunda release policy](https://camunda.com/release-policy/) with some specific clarifications which can be found below.
+Camunda 8 follows the [Camunda release policy](https://camunda.com/release-policy/) with the following specific clarifications.
+
+:::info
+You can find deprecation and support announcements on the [Announcements](announcements.md) page.
+:::
+
+![Stable and alpha channels when provisioning a cluster](./img/diagram-releases.png)
+
+## Alpha features and releases
+
+It is important to understand the different ways the term "alpha" is used in the context of Camunda releases and features.
+
+### Alpha feature
+
+Refers to a feature or component released as an alpha version, in an early state for you to test and participate in development by sharing your feedback before the feature reaches [general availability (GA)](/docs/reference/alpha-features.md#general-availability-ga). Some alpha features require turning on for your cluster before you can use them. See [alpha features](/docs/reference/alpha-features.md).
+
+### Alpha release
+
+Refers to a release made available between minor versions that allows you to preview an upcoming minor version and the alpha features included (for example, `8.6.0-alpha1`, `8.6.0-alpha2`, and so on). Camunda strives to release this type of release on a monthly basis. To learn more about the alpha features included in each alpha release, see [release notes](/docs/reference/release-notes/release-notes.md).
```

Comment:
This is super nit picky, but it feels odd that our example is far in the (minor) future from this version of the docs. I'm willing to ignore it now, but it would be cool if we could get some variable or reusable component into the docs to reference the current version (as it relates to the docs context). 🤔

Reply:
Great idea, I've changed this for each version so it is a better example as you say, a variable or reusable component sounds like an interesting idea definitely 👍